color_color.py

```diff
@@ -70,12 +81,9 @@ def make_acis_diagonal_rmf(arf):
     eb_hi = eb_lo + acis_gain
     eb_lo[0] = 0.001
 
-    _rmf = DataRMF("diagonal", detchans, elo, ehi, n_grp, f_chan,
+    return DataRMF("diagonal", detchans, elo, ehi, n_grp, f_chan,
```

Comment:
For future consideration - I really need to work on the difference between `DataRMF` and `RMF1D` and the like and describe where they are to be used. But that's for Sherpa, not here...